File: doc/src/scripts/Quaternion.cpp
#include <krpc.hpp>
#include <krpc/services/space_center.hpp>
#include <iostream>

using namespace krpc::services;

int main() {
  krpc::Client conn = krpc::connect();
  SpaceCenter sc(&conn);
  std::tuple<double,double,double,double> q = sc.active_vessel().flight().rotation();
  std::cout << std::get<0>(q) << " "
            << std::get<1>(q) << " "
            << std::get<2>(q) << " "
            << std::get<3>(q) << std::endl;
}

File: doc/src/scripts/Quaternion.cs
using System;
using System.Net;
using KRPC.Client;
using KRPC.Client.Services.SpaceCenter;

class QuaternionExample
{
    public static void Main ()
    {
        var connection = new Connection ();
        var spaceCenter = connection.SpaceCenter ();
        var vessel = spaceCenter.ActiveVessel;
        Tuple<double,double,double,double> q = vessel.Flight ().Rotation;
        Console.WriteLine (q.Item1 + "," + q.Item2 + "," + q.Item3 + "," + q.Item4);
    }
}
